src/views/v2/recent-past.php

```diff
@@ -39,25 +39,25 @@
 		data-view-breakpoint-pointer="<?php echo esc_attr( $breakpoint_pointer ); ?>"
 	<?php endif; ?>
 >
-	<div class="tribe-common-l-container tribe-events-l-container">
+	<div class="recent-past-events-container">
 		<?php $this->template( 'components/loader', [ 'text' => __( 'Loading...', 'the-events-calendar' ) ] ); ?>
 
-		<?php $this->template( 'components/json-ld-data' ); ?>
+<!--		<?php /*$this->template( 'components/json-ld-data' ); */?>
 
-		<?php $this->template( 'components/data' ); ?>
+		<?php /*$this->template( 'components/data' ); */?>
 
-		<?php $this->template( 'components/before' ); ?>
+		<?php /*$this->template( 'components/before' ); */?>
```

Comment:
* :no_entry_sign: Expected 1 space before closing PHP tag; 0 found
* :warning: This comment is 67% valid code; is this commented out code?

<sub>hash: 3a1416054c9ec5dfa8b29da9a7394eb0</sub>